README.md

```diff
@@ -8,4 +8,8 @@
 ![Swagger Validator](https://img.shields.io/swagger/valid/3.0?specUrl=https%3A%2F%2Fraw.githubusercontent.com%2FOAI%2FOpenAPI-Specification%2Fmaster%2Fexamples%2Fv2.0%2Fjson%2Fpetstore-expanded.json)
 [![GitHub license](https://img.shields.io/github/license/swissfintechinnovations/ca-payment)](https://github.com/swissfintechinnovations/ca-payment/blob/main/LICENSE)
 
-This is the official SFTI repo for the payment API. Documentations may be found in the [Wiki](https://github.com/swissfintechinnovations/ca-payment/wiki).
\ No newline at end of file
+This is the official SFTI repo for the payment API. Documentations may be found in the [Wiki](https://github.com/swissfintechinnovations/ca-payment/wiki).
+
+We refer to the SwaggerEditor for a graphical representation and source-file based documentation of the latest API state:
```

Comment:
May I suggest the following wording instead: "An easy-to-read representation of the Payment/XS2A APIs is accessible via the following links (please note that if you click on the following links, you will leave GitHub unless you use your browser's "open link in new tab/window" feature):"?